### PR TITLE
Addressed TimJ comments

### PR DESCRIPTION
**Pull request by gcomoretto**

### CODE REVIEW

File: body.tex [hunk 1]
```diff
@@ -325,4 +327,7 @@ \section{Conclusions and Outlook}
 This approach is not only used by DM, but also by other Rubin Observatory subsystems, that are facing the same challenge.
```
Comment (leannep) [inline body.tex]: Rubin Observatory subsystems that are facing the same challenge.

File: body.tex [hunk 2]
```diff
@@ -1,12 +1,14 @@
 \section{Introduction}
 
-The Rubin Observatory Data Management subystem, as described in \textit{Juric et al}\cite{2015arXiv151207914J},
-comprises software, services, and systems that will automatically reduce the raw data to produce science-ready data products.
+The Rubin Observatory will perform in its first 10 years of observations the Legacy Survey of Space and Time, as described
```
Comment (leannep) [inline body.tex]: In its first 10 years of observations Rubin Observatory will perform the Legacy Survey of Space and Time, as described in ....

File: body.tex [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
+in Ivezic et al\cite{2019ApJ...873..111I}. In the construction phase of the observatory, the Data Management subystem,
+described in Juric et al\cite{2015arXiv151207914J}, is in charge of developing and preparing
+software, services, and systems that will automatically reduce the raw data to produce science-ready data products.
```
Comment (leannep) [inline body.tex:6]: and preparing the software, services, and systems

**Review by leannep — CHANGES_REQUESTED** [review]

File: local.bib [hunk 4]
```diff
@@ -28,3 +28,199 @@ @inproceedings{10.1117/12.926797
...
+
+@misc{LSE-68,
+author =       {Gregory Dubois-Felsmann},
+title =        "{Camera Data Acquisition Interface, v2.4}",
+year =         2020,
+howpublished = {LSE-68, Rubin Observatory, \url{https://ls.st/LSE-68}},
+}
+
+@ARTICLE{2019ApJ...873..111I,
+       author = {{Ivezi{\'c}}, {\v{Z}}eljko and {Kahn}, Steven M. and
```
Comment (timj) [inline local.bib]: probably want this to be "FirstAuthor and others" to prevent lines and lines of names in the references.

File: body.tex [hunk 5]
```diff
@@ -322,7 +324,10 @@ \section{Conclusions and Outlook}
 or reusability of test cases are easy to implement and provide added value to the process.
 
 Finally, the verification control document provides a global overview of all DM requirements, ensuring we are not forgetting anything.
-This approach is not only used by DM, but also by other Rubin Observatory subsystems, that are facing the same challenge.
+This approach is not only used by DM, but also by other Rubin Observatory subsystems that are facing the same challenge.
 Using the same tools permit everybody to use the same template and have the same documentation format across the project.
 
+\acknowledgments
+This material is based on work supported in part by the  {National Science Foundation} through Cooperative Agreement 1258333 managed by the  {Association of Universities for Research in Astronomy} ( {AURA}), and the  {Department of Energy} under  {Contract} No.  {DE}-AC02-76SF00515 with the  {SLAC} National Accelerator Laboratory. Additional LSST funding comes from private donations, grants to universities, and in-kind support from LSSTC Institutional Members.
```
Comment (timj) [inline body.tex]: There seems to be an extra space before "AURA".

**Review by leannep — APPROVED** [review]

**Pull request merged**